packages/internal/test-util/src/lib/error-throwing-driver/error-throwing-driver-root.module.ts

```diff
@@ -1,40 +1,22 @@
-import { NgModule, Optional, SkipSelf } from '@angular/core';
+import { inject, NgModule, InjectFlags } from '@angular/core';
 
-import {
-  LumberjackLogDriverConfig,
-  lumberjackLogDriverConfigToken,
-  lumberjackLogDriverToken,
-} from '@ngworker/lumberjack';
+import { lumberjackLogDriverToken } from '@ngworker/lumberjack';
 
-import { errorThrowingDriverConfigToken } from './error-throwing-driver-config.token';
-import { ErrorThrowingDriverConfig } from './error-throwing-driver.config';
 import { ErrorThrowingDriver } from './error-throwing.driver';
 
-export function errorThrowingDriverFactory(
-  logDriverConfig: LumberjackLogDriverConfig,
-  errorThrowingDriverConfig: ErrorThrowingDriverConfig
-): ErrorThrowingDriver {
-  const config: ErrorThrowingDriverConfig = {
-    ...logDriverConfig,
-    ...errorThrowingDriverConfig,
-  };
-
-  return new ErrorThrowingDriver(config);
-}
-
 @NgModule({
   providers: [
     {
-      deps: [lumberjackLogDriverConfigToken, errorThrowingDriverConfigToken],
       provide: lumberjackLogDriverToken,
-      useFactory: errorThrowingDriverFactory,
+      useClass: ErrorThrowingDriver,
       multi: true,
     },
   ],
 })
 export class ErrorThrowingDriverRootModule {
-  constructor(@Optional() @SkipSelf() maybeNgModuleFromParentInjector?: ErrorThrowingDriverRootModule) {
-    if (maybeNgModuleFromParentInjector) {
+  maybeNgModuleFromParentInjector = inject(ErrorThrowingDriverRootModule, InjectFlags.SkipSelf | InjectFlags.Optional);
```

Comment:
nit: separate property and constructor with a blank line.

Reply:
issue: mark as private.
```suggestion
  private maybeNgModuleFromParentInjector = inject(ErrorThrowingDriverRootModule, InjectFlags.SkipSelf | InjectFlags.Optional);
```